src/Models/FieldMaps/b1_afi.m

```diff
@@ -0,0 +1,92 @@
+classdef b1_afi < AbstractModel & FilterClass
```

Comment:
Since this is a field mapping module, I added inheritance to the FilterClass in 126a14e. This required some refactoring too, and I ended up removing the code that was written for your attempt to parallelisation (but wasn't working yet). We could save that code in another branch or in a [Gist](https://gist.github.com) to be used in the future in another model like qMT, however as we discussed initially the end goal for this model was always to vectorize it for speed. Thanks for developping an attempt to parrallelize it though - this simple use case will be simpler to explore than the more complicated qMT code!

Reply:
> 
> 
> @jvelazquez-reyes I think the model should work well "out-of-the-box" now. Please test it once more, and if you're also satisfied with it, we'll assign a reviewer for this PR.

@mathieuboudreau I did the manual GUI tests and ticked the checkboxes when applicable. I tested it a couple of times and everything seems to work fine. 

For the interactive figures for the blog post. Do you have any template I should follow (as you said you have done several examples)?

Reply:
Let's schedule a video call by email to discuss the blog post. I don't have clear instructions, we'll have to base ourselves on a recent one I did and then adapt it for your model, and I think it'll be smoother if we walk through some of the major steps together (so that I can also remind myself of what I did haha).